### PR TITLE
Feature/getinvolved page

### PR DESCRIPTION
**Pull request by muhammadqamar**
## Summary

Adding Two Pages (Articles and Get Involved-A)
Apply Hexagon Grid in Different Pages

## Quality Checklist

- [ ] New features are fully tested (No matter how much Coverage Karma you have)
- [ ] **[v0.2 onward]** New features are included in the documentation
- [ ] **[v0.2 onward]** Breaking changes are documented with clear instructions of what 

### Coverage Karma

- [ ] If your PR decreases test coverage, do you feel you have built enough `Coverage Karma`* to justify it?

*Coverage Karma can be earned by disproportionally increasing test coverage in the rest of your contributions.
It's an honesty policy - you keep count of your own.

### CODE REVIEW

File: src/components/sectionelements/helpsection/HelpSection.jsx [hunk 1]
```diff
@@ -91,7 +111,8 @@ const HelpSection = () => {
               button="Enter the library"
             />
             <Box className={classes.hexagonBox}>
-              <TripleHexagon image={Image} />
+              {/* <TripleHexagon image={Image} /> */}
```
Comment (thclark) [inline src/components/sectionelements/helpsection/HelpSection.jsx:114]: Remove the comment. We have a shiny new HexagonGrid now ;)

File: src/components/sectionelements/sectionwithhexagon/AboutHexagonSection.jsx [hunk 2]
```diff
@@ -71,7 +95,8 @@ const AboutHexagonSection = () => {
               <SectionHeading text="Do more with your data" />
             </Box>
             <Box className={classes.hexagonBox}>
-              <MultipleHexagonWithImage image={Image} />
+              {/* <MultipleHexagonWithImage image={Image} /> */}
```
Comment (thclark) [inline src/components/sectionelements/sectionwithhexagon/AboutHexagonSection.jsx:98]: remove comment

File: src/components/sectionelements/sectionwithhexagon/consultancyHexagonsections/DigitalHexagonSection.jsx [hunk 3]
```diff
@@ -113,7 +137,8 @@ const DigitalHexagonSection = () => {
           </Box>
           <Box className={classes.HexagonDetailBox}>
             <Box className={classes.hexagonBox}>
-              <MultipleHexagonWithImage image={Image} />
+              {/* <MultipleHexagonWithImage image={Image} /> */}
```
Comment (thclark) [inline src/components/sectionelements/sectionwithhexagon/consultancyHexagonsections/DigitalHexagonSection.jsx:140]: comment

File: src/components/sectionelements/sectionwithhexagon/consultancyHexagonsections/StrategyHexagonSection.jsx [hunk 4]
```diff
@@ -89,7 +112,8 @@ const StrategyHexagonSection = () => {
           </Box>
           <Box className={classes.HexagonDetailBox}>
             <Box className={classes.hexagonBox}>
-              <ReverseMultipleHexagonImage image={Image} />
+              {/* <ReverseMultipleHexagonImage image={Image} /> */}
```
Comment (thclark) [inline src/components/sectionelements/sectionwithhexagon/consultancyHexagonsections/StrategyHexagonSection.jsx:115]: comment

File: src/components/sectionelements/sectionwithhexagon/consultancyHexagonsections/SupportHexagonSection.jsx [hunk 5]
```diff
@@ -96,7 +118,8 @@ const SupportHexagonSection = () => {
           </Box>
           <Box className={classes.HexagonDetailBox}>
             <Box className={classes.hexagonBox}>
-              <MultipleHexagonWithImage image={Image} />
+              {/* <MultipleHexagonWithImage image={Image} /> */}
```
Comment (thclark) [inline src/components/sectionelements/sectionwithhexagon/consultancyHexagonsections/SupportHexagonSection.jsx:121]: comment

**Review by thclark — CHANGES_REQUESTED** [review]
Main change is a need for pagination on the articles page

File: src/components/sectionelements/articlessection/LatestArticlesSection.jsx [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+              </Select>
+            </Box>
+            <Box className={classes.showingBox}>
+              <Typography
+                variant="body1"
+                color="textSecondary"
+                component="span"
+              >
+                Showing 6 of 16
+              </Typography>
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/articlessection/LatestArticlesSection.jsx:214]: How many articles will display in this page

[the remark this answers is not in the document]
Comment (thclark) [reply src/components/sectionelements/articlessection/LatestArticlesSection.jsx:214]: Let’s take the lead from the designer and put 6 articles on the page

**Pull request merged**